clients/windows/src/input/mod.rs

```diff
@@ -5,5 +5,4 @@ pub mod message;
 pub mod mouse;
 pub mod pointer;
 
-pub use message::Message;
 pub use message::Point;
```

Comment:
Is this even worth keeping? I think the only place this is taken advantage of is here
https://github.com/lockbook/lockbook/blob/e92ed51631ff75ce67ea2527437c6f395dde0831/clients/windows/src/input/mouse.rs#L4